imod/mf6/ims.py

```diff
@@ -419,7 +425,9 @@ def __init__(
             "scaling_method": scaling_method,
             "reordering_method": reordering_method,
             "print_option": print_option,
-            "csv_output": csv_output,
+            "outer_csvfile": outer_csvfile,
+            "inner_csvfile": inner_csvfile,
+            "ats_outer_maximum_fraction": ats_outer_maximum_fraction,
             "no_ptc": no_ptc,
         }
         # Make sure the modelnames are set as a variable rather than dimension:
```

Comment:
I think we should have some validation to check if no_ptc equals"first" or "all". I don't wanna burden you further with defining a ValidationScheme for this, but could you add some code in the init that does:

```suggestion
            if setting.lower() in ["first", "all"]:
                return setting.lower()
            else:
                raise ValueError(
                    f"Received wrong setting for no_ptc. String should be one of ['first', 'all'], instead got {setting}"
                )
        # Make sure the modelnames are set as a variable rather than dimension:
```

(Modified from ``OutputControl._get_ocsetting``)

Reply:
I've added an OptionSchema instead. Should be generally useful.